docs/ECCTOOL.md

```diff
@@ -16,11 +20,262 @@ The following subcommands are available:
+| Id                                   | Host Id                              | Keyspace              | Table              | Status   | Repaired(%) | Completed at |
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------
+| 497eb4cf-9275-4216-9cca-12958bde28af | 6424a5fa-69ea-49a3-a542-4751d0283c9a | test2                 | table2             | IN_QUEUE | 0.00        | -            |
+| 497eb4cf-9275-4216-9cca-12958bde28af | 045c01c1-ff50-4de2-8da3-1b9270c382b5 | test2                 | table2             | IN_QUEUE | 0.00        | -            |
+| ea32c8b5-3e8a-466a-b5f5-f9248e73774c | 6424a5fa-69ea-49a3-a542-4751d0283c9a | test                  | table2             | IN_QUEUE | 0.00        | -            |
+| ea32c8b5-3e8a-466a-b5f5-f9248e73774c | 045c01c1-ff50-4de2-8da3-1b9270c382b5 | test                  | table2             | IN_QUEUE | 0.00        | -            |
+| 0d8845fd-84dc-435c-8b8b-83b701dd2cbd | 045c01c1-ff50-4de2-8da3-1b9270c382b5 | keyspaceWithCamelCase | tableWithCamelCase | IN_QUEUE | 0.00        | -            |
+| 0d8845fd-84dc-435c-8b8b-83b701dd2cbd | 6424a5fa-69ea-49a3-a542-4751d0283c9a | keyspaceWithCamelCase | tableWithCamelCase | IN_QUEUE | 0.00        | -            |
+| c5f830b0-533a-464f-80ed-aa8b90248ba3 | 6424a5fa-69ea-49a3-a542-4751d0283c9a | test2                 | table1             | IN_QUEUE | 0.00        | -            |
+| c5f830b0-533a-464f-80ed-aa8b90248ba3 | 045c01c1-ff50-4de2-8da3-1b9270c382b5 | test2                 | table1             | IN_QUEUE | 0.00        | -            |
+| 73c27554-58a5-47b9-a2ab-01b9fcfad4f0 | 6424a5fa-69ea-49a3-a542-4751d0283c9a | test                  | table1             | IN_QUEUE | 0.00        | -            |
+| 73c27554-58a5-47b9-a2ab-01b9fcfad4f0 | 045c01c1-ff50-4de2-8da3-1b9270c382b5 | test                  | table1             | IN_QUEUE | 0.00        | -            |
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------
+Summary: 0 completed, 10 in queue, 0 blocked, 0 warning, 0 error
+```
+
+Looking at the example output above, the columns are:
+
+`Id` - the manual repair ID, manual repair triggered on several hosts will have the same ID.
+
+`Host Id` - the host id of the Cassandra instance ecChronos responsible for performing manual repair is connected to.
+
+`Keyspace` - the keyspace the manual repair is run on.
+
+`Table` - the table the manual repair is run on.
+
+`Status` - the status of the manual repair. This will always be `IN_QUEUE` for newly run manual repairs.
+
+`Repaired(%)` - the number of ranges repaired vs total ranges.
+This value should never go down. This will always be `0` for newly run manual repairs.
```

Comment:
Same as above.

For repairs, this value should never go down.
Or
Unlike Schedules, this value should never go down.